packages/cli/src/commands/generate/sdl/__tests__/sdl.test.js

```diff
@@ -1,69 +1,124 @@
 global.__dirname = __dirname
 
 import { loadGeneratorFixture } from 'src/lib/test'
+import { getDefaultArgs } from 'src/lib'
 
 import * as sdl from '../sdl'
 
 afterEach(() => {
   jest.clearAllMocks()
 })
 
-test('returns exactly 3 files', async () => {
-  const files = await sdl.files({ name: 'Post', crud: false })
+const extensionForBaseArgs = (baseArgs) =>
+  baseArgs && baseArgs.typescript ? 'ts' : 'js'
 
-  expect(Object.keys(files).length).toEqual(3)
-})
+const itReturnsExactlyThreeFiles = (baseArgs = {}) => {
```

Comment:
The reason I'm doing this is to avoid duplicating the same set of tests within typescript and javascript description blocks. I'd be happy to employ another solution here. (Alternatively we just duplicate the tests, but that feels slightly brittle to me)

Reply:
(Although, to argue against myself, I just duped test calls for the scaffold generator in two different `description` blocks...)